patch/0001-prebuild.patch

```diff
@@ -0,0 +1,27 @@
+From ca6fc1ec27e4e79cd15441548f4b700a658a0ae5 Mon Sep 17 00:00:00 2001
+From: neovali <valigarmanda55@gmail.com>
+Date: Fri, 10 Nov 2023 11:27:41 +0800
+Subject: [PATCH] prebuild
+
+---
+ Extension/Lua/{lua.pro => Lua.pro} | 0
+ lib/x64/linux/liblua53.a           | 2 +-
+ 2 files changed, 1 insertion(+), 1 deletion(-)
+ rename Extension/Lua/{lua.pro => Lua.pro} (100%)
+
+diff --git a/Extension/Lua/lua.pro b/Extension/Lua/Lua.pro
+similarity index 100%
+rename from Extension/Lua/lua.pro
+rename to Extension/Lua/Lua.pro
+diff --git a/lib/x64/linux/liblua53.a b/lib/x64/linux/liblua53.a
+index d5dc920..bf39146 120000
+--- a/lib/x64/linux/liblua53.a
++++ b/lib/x64/linux/liblua53.a
```

Comment:
does this work on aarch64?

Reply:
no

Reply:
can't this be just built?

Reply:
it seems that [upstream](https://github.com/KikoPlayProject/KikoPlay/tree/master/lib/x64) only provides x64 libraries

Reply:
fine, the author said "I have never tried it before." 
maybe i works